src/components/Homepage/LendByCategory/FeaturedLoansCarousel.vue

```diff
@@ -1,13 +1,9 @@
 <template>
 	<div>
 		<h3>
-			{{ categoryPreLink(currentSlideCategory.id) }}
-			<router-link
-				:to="currentSlideCategory.url"
-			>
-				{{ cleanCategoryName(currentSlideCategory) }}
-			</router-link>
-			{{ categoryPostLink(currentSlideCategory.id) }}
+			<a :href="currentSlideCategory.url">
```

Comment:
Curious why you changed from router-link here

Reply:
@ryan-ludwig the API returns an absolute link. router-link can only be used for relative links